lib/smart_proxy_ansible/validate_settings.rb

```diff
@@ -0,0 +1,7 @@
+module Proxy::Ansible
+  class ValidateSettings < ::Proxy::PluginValidators::Base
+    def validate!(settings)
+      raise NotExistingWorkingDirException.new("Working directory does not exist") unless File.expand_path(settings[:working_dir]).nil? || File.directory?(File.expand_path(settings[:working_dir]))
```

Comment:
Maybe I've already asked, but why do we expand the path and then ask if that is `nil?` in the first condition?

Reply:
@ezr-ondrej  I added the expand_path because Adam corrected me in a previous comment and if I am not mistaken we are creating a tmp directory if the :working_dir is nil https://github.com/theforeman/smart_proxy_ansible/blob/cd051f6859494e7a70a250696fc10d7276229d9a/lib/smart_proxy_ansible/runner/ansible_runner.rb#L230

Reply:
> I just need to ask if the :working dir is nil? and the expand_path is redundant?

That's correct.

Reply:
@adamruzicka, @ezr-ondrej  Fixed. Thank you for the feedback